lib/dispatcher/index.js

```diff
@@ -37,27 +41,34 @@ module.exports = ({ schema, emailer, logger, publicUrl }) => async ({ task, noti
       taskType: get(content, `type[${licenceType}][${taskType}]`),
       identifier: get(content, `identifier[${licenceType}][${taskType}]`),
       identifierValue: getIdentifier({ model, licenceType, applicant }),
-      prevStatus: get(content, `status[${task.meta.previous}]`),
-      newStatus: get(content, `status[${task.meta.next}]`),
       taskUrl: `${publicUrl}/tasks/${task.id}?notification=${task.id}`,
       licenceNumber: model && model.licenceNumber,
       licenceUrl: `${publicUrl}/${licencePath}?notification=${task.id}`,
       loginUrl: publicUrl,
       ...notification
     };
 
+    const subject = mustache.render(subjectTemplate, templateVars);
+
+    if (task.meta) {
+      Object.assign(templateVars, {
+        prevStatus: get(content, `status[${task.meta.previous}]`),
+        newStatus: get(content, `status[${task.meta.next}]`)
+      });
+    }
+
     return Promise.resolve()
       .then(() => loadTemplate(templateName, templateVars))
       .then(html => {
-        return emailer('/custom', {
-          method: 'POST',
-          json: {
-            to: recipient.email,
-            name: recipientName,
-            subject,
-            html
-          }
-        });
+        const params = {
+          profileId: recipient.id,
+          to: recipient.email,
+          name: recipientName,
+          identifier: notification.identifier || task.id,
```

Comment:
Worth checking here, is the `task` actually the task, or is it the activity event?

The identifier needs to be the activity event, so that the same task can notify the same user multiple times, but each event can only notify each user once.